extra-network/cgproxy/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=cgproxy
+PKGSEC=net
+PKGDEP="libbpf iproute2 which iptables"
```

Comment:
Why does it depend on `which`? Is this a script?

Reply:
It uses `which` to find the command(write in noproxy) directory.